extension-androidauto/CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog for Mapbox Maps Android Auto Extension SDK
+
+Mapbox welcomes participation and contributions from everyone.
+
+# main
+## Features ✨ and improvements 🏁
+## Bug fixes 🐞
+
+# extension-androidauto-v0.1.0-beta.1 April 5, 2022
```

Comment:
Also I'm not sure if we want to have a beta.1 release, I'd suggest we start with v0.1.0 release. wdyt cc @tobrun @hermu ?

Reply:
As agreed separately with the team, we'd go with `extension-androidauto-v0.1.0`